<launch>	


  <node pkg="interactive_walk_planner" type="cloud_assembler_ocu" name="cloud_assembler_ocu" output="screen">
    </node>

  <node pkg="interactive_walk_planner" type="foot_interface" name="foot_interface" output="screen">
    </node>

  <node pkg="interactive_walk_planner" type="step_plan" name="step_plan" output="screen">
  <param name="action_path"  value="$(find interactive_walk_planner)/action_sets/" />
    </node>


</launch>
